src/status_im/contexts/wallet/wallet_connect/responding_events.cljs

```diff
@@ -157,8 +196,26 @@
 ;; - Failed "responding" (signing or sending message/transaction)
 (rf/reg-event-fx
  :wallet-connect/reject-session-request
- (fn [_ _]
-   {:fx [[:dispatch
-          [:wallet-connect/send-response
-           {:error (wallet-connect/get-sdk-error
-                    constants/wallet-connect-user-rejected-error-key)}]]]}))
+ (fn [{:keys [db]} _]
+   (let [event              (get-in db [:wallet-connect/current-request :event])
```

Comment:
I know you know 🙊 but we are accumulating a certain tech debt due to the lack of tests in some events. Adding tests later misses the point that tests exist to support changes, that's basically why they exist.

If, during development, we are able to test a non-trivial event manually, then that also means we can also simply connect to a REPL, capture the input & output and transform that into an event test using `deftest-event`. That shouldn't slow us down too much and would protect us to catch things QAs usually can't realistically verify.

Reply:
pinky promise as soon as we have the bugs fixed for the release candidate, we'll start adding tests. We even have an issue for [this](https://github.com/status-im/status-mobile/issues/20776) to show our commitment to testing 😄

Reply:
> have an issue for https://github.com/status-im/status-mobile/issues/20776 to show our commitment to testing 😄

hauahauha ❤️